samples/snippets/csharp/tutorials/patterns/finished/toll-calculator/TollCalculator.cs

```diff
@@ -34,42 +34,90 @@ vehicle switch
                 DeliveryTruck t when (t.GrossWeightClass < 3000) => 10.00m - 2.00m,
                 DeliveryTruck t => 10.00m,
 
-                { }  => throw new ArgumentException(message: "Not a known vehicle type", paramName: nameof(vehicle)),
+                { } => throw new ArgumentException(message: "Not a known vehicle type", paramName: nameof(vehicle)),
                 null => throw new ArgumentNullException(nameof(vehicle))
             };
 
         // <SnippetFinalTuplePattern>
         public decimal PeakTimePremium(DateTime timeOfToll, bool inbound) =>
             (IsWeekDay(timeOfToll), GetTimeBand(timeOfToll), inbound) switch
             {
-                (true, TimeBand.Overnight,   _)     => 0.75m,
-                (true, TimeBand.Daytime,     _)     => 1.5m,
-                (true, TimeBand.MorningRush, true)  => 2.0m,
+                (true, TimeBand.Overnight, _) => 0.75m,
+                (true, TimeBand.Daytime, _) => 1.5m,
+                (true, TimeBand.MorningRush, true) => 2.0m,
                 (true, TimeBand.EveningRush, false) => 2.0m,
-                (_,    _,                    _)     => 1.0m,
+                (_, _, _) => 1.0m,
```

Comment:
Could this just be a single `_` instead?
```suggestion
                _ => 1.0m,
```